portfolio/src/main/webapp/index.html

```diff
@@ -8,16 +8,30 @@
   </head>
     <header>
       <h1>JENNIFER JIMENEZ</h1>
+      <div class="logos">
+        <a href="https://www.linkedin.com/in/jen-jimenez" target="_blank"> 
+          <img src="/images/linkedin.png" id="logo"/>
+        </a>
+        <a href="https://github.com/jennifer-jimenez" target="_blank"> 
+          <img src="/images/git.png" id="logo"/>
+        </a>
+        <a href="https://www.instagram.com/prettyoddjen/" target="_blank"> 
+          <img src="/images/ig.png" id="logo"/>
+        </a>
+      </div>
     </header>
     <nav>
       <ul class="navbar">
-        <li>Home</li>
-        <li>Projects & Work</li>
-        <li>Gallery</li>
+        <li><a href="index.html" class="nav_link"><u>Home</u></a></li>
```

Comment:
It's probably best to omit the `<u>` tag here, especially if it's used just for the underline style (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u#usage_notes). You can use CSS to select and underline elements using the [`text-decoration`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration) property, and JS to manipulate classes that apply that style on the appropriate elements if you want the current page header to always be underlined, for instance.